lib/fs-wrapper.js

```diff
@@ -87,6 +90,20 @@ function mkdirpSync(dir) {
   fs.mkdirSync(dir, {recursive: true});
 }
 
+/**
+ * Copy files from srcFolder to destFolder.
+ * @param {Path} srcFolder
+ * @param {Path} destFolder
+ * @param {Path[]} files
+ * @returns {Promise<void>}
+ */
+function copyFiles(srcFolder, destFolder, files) {
+  const promises = files.map((file) =>
+    copyFile(path.join(srcFolder, file), path.join(destFolder, file))
+  );
+  return Promise.all(promises).then(() => undefined);
```

Comment:
Why not this?

```suggestion
  return Promise.all(promises);
```

Maybe I've forgotten something, but the noop shouldn't be necessary.

Reply:
I think it's to have the return type `@returns {Promise<void>}` instead of `@returns {Promise<void[]>}` or `@returns {Promise<something[]>}`

Reply:
> I think it's to have the return type `@returns {Promise<void>}` instead of `@returns {Promise<void[]>}` or `@returns {Promise<something[]>}`

Hmm. That makes sense. I would think that `void[]` is assignable to `void`, but maybe not.